wls-gui-admintool/src/components/wahltag/BaseDialogWahltagOverrideWahlterminConfirmation.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-dialog
+    :model-value="visible"
+    data-test="dialog-wahltag-override-wahltermindaten-confirmation"
+  >
+    <v-card>
+      <v-card-title>Wahltermindaten erneut anlegen</v-card-title>
+      <v-card-text>
+        <div data-test="div-confirm-information">
+          Für diesen Wahltag existieren bereits Wahltermindaten. Bitte geben sie
+          "{{ requiredConfirmText }}" in das Eingabefeld ein und bestätigen sie
+          um die alten Wahltermindaten zu löschen und neue zu erstellen.
+        </div>
+        <v-text-field
+          v-model="confirmText"
+          ref="confirmTextField"
+          :label="confirmTextFieldLabel"
+          :rules="[requiredText(requiredConfirmText)]"
+          class="mt-2"
```

Comment:
```suggestion
          class="mt-5"
```
sieht dann nicht ganz so gequetscht aus

Reply:
bei 2 belassen da der Abstand doch im Moment so passt